python/az/aro/azext_aro/custom.py

```diff
@@ -60,6 +61,8 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
         rp_client_id = os.environ['AZURE_FP_CLIENT_ID']
 
     rp_client_sp = aad.get_service_principal(rp_client_id)
+    if rp_client_sp is None:
```

Comment:
This seems to only make sense in the local host case.  Can we only check it in that case?